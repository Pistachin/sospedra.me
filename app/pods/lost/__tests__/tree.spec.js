/* global describe, expect, it */
import React from 'react'
import { shallow } from 'enzyme'

import Lost from '../index'

describe('Lost wrapper suite', () => {
  const wrapper = shallow(<Lost />)

  it('should have a text with the pod name', () => {
    expect(wrapper.find('p').text()).toBe('Lost')
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
