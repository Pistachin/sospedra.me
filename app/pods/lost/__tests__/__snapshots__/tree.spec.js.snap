exports[`Home wrapper suite should match the snapshot 1`] = `
<div
  id="content">
  <h1>
    Â 
  </h1>
  <h2>
    Welcome!
  </h2>
  <ul>
    <li
      className="link_13o8hyc">
      <a
        href="http://brunch.io">
        Brunch homepage
      </a>
    </li>
    <li
      className="link_13o8hyc">
      <a
        href="https://facebook.github.io/react/">
        React.js homepage
      </a>
    </li>
  </ul>
</div>
`;
